src/main/java/uk/gov/pay/webhooks/deliveryqueue/managed/SendAttempter.java

```diff
@@ -70,8 +72,15 @@ public void attemptSend(WebhookDeliveryQueueEntity queueItem) {
         var retryCount = webhookDeliveryQueueDao.countFailed(queueItem.getWebhookMessageEntity());
         Instant start = instantSource.instant();
 
+        URL url = null;
+        try {
+            url = new URL(webhook.getCallbackUrl());
+        } catch (MalformedURLException e) {
+            LOGGER.info(format("Callback URL %s is malformed.", webhook.getCallbackUrl()));
```

Comment:
```suggestion
            LOGGER.info("Callback URL {} is malformed.", webhook.getCallbackUrl());
```
You don't need the string format here, SLF4J can handle the param substitution